### DISCUSSION TITLE
CLI - Group by tag / category

### DISCUSSION THREAD

**Discussion by newadbie** [Infisical/infisical#5631] [Feature Request]
### Feature description
Extend CLI by command that allow user to export secrets grouped by tags, or maybe even new field 'category' because category can be only one. Using this command would export values in order, example below:

```ENV
#system
PORT=4000
APP_URL=http://localhost:4000/

#mail-service
MAIL_SERVICE_URL=http://localhost:8000/
MAIL_SERVICE_TOKEN=MyV$RyS$CR$Token

#cms
CMS_URL=http://localhost:3000/
CMS_TOKEN=secret_ToKeN
```

### Why would it be useful?
For users like my it would be very useful because i prefer to work with .env file locally, i am using infisical because i am working in team and envs are changing all time, but i prefer to know which envs has been added (we are using t3 env) and in which branch so i am pulling envs every time it is only necessary, to not avoid changes in application i will missunderstand.
I am not using UI to override just i am overriding envs in my local file, and grouping envs by category / tag would make this work more efficient

**Comment by akhilmhdh** [comment]
Hey @maidul98 is this still open. If so @quinton11 would like to take this issue.
